sdk/formrecognizer/ai-form-recognizer/swagger/FormRecognizer.yml

```diff
@@ -0,0 +1,1666 @@
+---
```

Comment:
what is the purpose of this file?

Reply:
It is the swagger used to generate the autorest layer.

Reply:
why not using the upstream swagger instead?

Reply:
It is not merged upstream yet.